package/MDAnalysis/core/selection.py

```diff
@@ -156,6 +156,16 @@ def apply(self, group):
 
         return group.universe.atoms[idx]
 
+def return_empty_on_apply(func):
```

Comment:
Iirc this kills docstrings so there's a `functools.decorate` or something to preserve docstrings after wrapping